### ISSUE TITLE
Does Asahi Linux support installing OS on USB?

### ISSUE THREAD

**Issue by LincolnKermit** [AsahiLinux/docs#126]
Hi, I wanted to know if Asahi Linux can be installed on a flashdrive.

Macbook user does not have a lot of storage...

**Comment by rowanG077** [comment]
No

**Comment by eleanordoesntcode** [comment]
> See [AsahiLinux/asahi-installer#205](https://github.com/AsahiLinux/asahi-installer/issues/205)
> 
> I would never recommend it. It's a path full of mines. You better know what you are doing.

this guide is for Arch Asahi Linux. Most of the advancements are being done on Fedora right now so it'd be better to have a Fedora guide

**Comment by LincolnKermit** [comment]
Yes but since its not an ISO file but a bash installer, i was asking.
